src/EnergyPlus/OutputProcessor.hh

```diff
@@ -74,6 +74,28 @@ struct EnergyPlusData;
 
 namespace OutputProcessor {
 
+    // enumerations for SetupOutputVariable calls -- for now these are direct mappings from the original strings, no change in functionality
+    enum class SOVTimeStepType
+    {
+        Invalid = -1,
+        System,
+        HVAC,
+        Zone,
+        Plant,
+        Num
+    };
+    constexpr std::array<std::string_view, (int)SOVTimeStepType::Num> sovTimeStepTypeStrings = {"System", "HVAC", "Zone", "Plant"};
+    enum class SOVStoreType
+    {
+        Invalid = -1,
+        State,
+        NonState,
+        Summed,
+        Average,
+        Num
+    };
+    constexpr std::array<std::string_view, (int)SOVStoreType::Num> sovStoreTypeStrings = {"State", "NonState", "Summed", "Average"};
```

Comment:
New enumerations that are based directly on the original strings passed in.  In a future effort, we need to evaluate whether these should be kept or not, and try to move to the StoreType and TimeStepType enums below in this file.  However, if we remove those, then we lose the ability to differentiate between System, HVAC, Zone, and Plant on the TimeStepType, and those are still seen user-side in the SQL output indexGroup.

Reply:
Well, OK for now, but indexGroup is a completely different animal from TimeStepType and StoreType. TimeStepType should only be zone or HVAC (or system of whatever  we choose). indexGroup is a larger set with a few common names. Perhaps these were sharing the same list before, so when it was reduced it impacted both parameters.